components/semotus/HISTORY.md

```diff
@@ -1,3 +1,5 @@
+## 3.2.1
```

Comment:
This is not a defect, right, may be we can go with 3.3.0

Reply:
I chose to mark it as a bug fix because it's a dev dependency audit fix, so it's fixing existing vulns.